packages/smithy-core/src/smithy_core/deserializers.py

```diff
@@ -247,6 +264,9 @@ def read_document(self, schema: "Schema") -> "Document":
     def read_timestamp(self, schema: "Schema") -> datetime.datetime:
         self._invalid_state(schema)
 
+    def read_data_stream(self, schema: "Schema") -> "_Stream":
```

Comment:
What's the reason we're importing `StreamingBlob` as a private variable? It seems odd that we're returning a private type from a public interface.

Reply:
I really don't want these being re-exported since there's two symbols of the same name in different namespaces. It's not a major deal though. From a type checking perspective it'll expand the union instead of referring to the local symbol name anyway.

Reply:
Oh wait they're in TYPE_CHECKING so won't be exported in any capacity